scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# Wait until the height of the block is greater than or equal to 1.
+h="0"
+until [[ ${h} == '"1"' ]]
+do
+  h=$(fnsad query block 1 --node http://localhost:26658 --chain-id simd-testing --keyring-backend test | jq .block.header.height)
+done
+
+# send transaction to node
+cur_path=`pwd`
+
+artifacts_path="artifacts"
+
+## parameters
+CONTRACT_ADDRESS="link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"
+FROM_ACCOUNT="link146asaycmtydq45kxc8evntqfgepagygelel00h"
+URL="http://localhost:26658"
+CODE_ID=1
+
+# This is a function that checks if the result is as expected.
```

Comment:
this looks simply checking the result has the word "failed"

Reply:
Fixed here
[chore: modify the comment about check_result()](https://github.com/Finschia/finschia-wasm/pull/3/commits/26b66d739d7c9937a1cbb07e7499b5194963ba15)